cmd/build-extensions-container.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"github.com/coreos/coreos-assembler/internal/pkg/cosash"
+	"path/filepath"
+	"io/ioutil"
+	"crypto/sha256"
+	"io"
+	"log"
+	"os"
+	"strings"
+	"encoding/json"
+	//"os/exec"
+)
+
+type MetaJSON struct {
```

Comment:
We should have all this defined in `"github.com/coreos/coreos-assembler-schema/cosa"`

Reply:
all makes sense now...